src/events.js

```diff
@@ -671,57 +671,43 @@ export const addEventCheckInCompleteForm = (isCheckedIn, checkOutFlag) => {
                         let collection = response.data.filter(res => res['331584571'] == visit.concept);
                         if (collection.length === 0) continue;
 
-                        const confirmRepeat = await swal({
-                            title: "Warning - Participant Previously Checked In",
-                            icon: "warning",
-                            text: "Participant " + data['399159511'] + " " + data['996038075'] + " was previously checked in on " + new Date(data['331584571'][visit.concept]['840048338']).toLocaleString() + " with Collection ID " + collection[0]['820476880'] + ".\r\nIf this is today, DO NOT check the participant in again.\r\nNote Collection ID above and see Check-In SOP for further instructions.\r\n\r\nIf this is is not today, you may check the participant in for an additional visit.",
-                            buttons: {
-                                cancel: {
-                                    text: "Cancel",
-                                    value: "cancel",
-                                    visible: true,
-                                    className: "btn btn-danger",
-                                    closeModal: true,
-                                },
-                                confirm: {
-                                    text: "Continue with Check-In",
-                                    value: 'confirmed',
-                                    visible: true,
-                                    closeModal: true,
-                                    className: "btn btn-success",
-                                }
-                            }
-                        });
 
+                        // Create a function to show the Bootstrap modal
+const showConfirmRepeatModal = () => {
+    const participantData = {
+        name: data['399159511'] + ' ' + data['996038075'],
+        checkInDate: new Date(data['331584571'][visit.concept]['840048338']).toLocaleString(),
+        collectionID: collection[0]['820476880'],
+    };
+    
+    $('#participantInfo').text('Participant ' + participantData.name);
+    $('#checkInDate').text(participantData.checkInDate);
+    $('#collectionID').text(participantData.collectionID);
+    $('#confirmRepeatModal').modal('show'); 
+    $('#continueCheckIn').on('click', function() {
+    $('#confirmRepeatModal').modal('hide'); 
+    });
+};
+
+    const confirmRepeat = showConfirmRepeatModal();
```

Comment:
confirmRepeat is not going to receive a value because showConfirmRepeatModal() does not return a value.  And with no await this code is always going to run the checkInParticipant regardless of the user interaction